### ISSUE TITLE
Feature request: separate preview

### ISSUE THREAD

**Issue by darsnack** [notaapp/issues#56]
Have you considered side-by-side preview and preview in a separate window? I've attached examples of both below.

Side-by-side (from iA Writer):
<img width="1641" alt="Screen Shot 2021-05-16 at 5 28 05 PM" src="https://user-images.githubusercontent.com/4811350/118414761-1598f100-b66c-11eb-80c7-3d18bed3d708.png">

External (from Ulysses):
<img width="1767" alt="Screen Shot 2021-05-16 at 5 29 02 PM" src="https://user-images.githubusercontent.com/4811350/118414793-382b0a00-b66c-11eb-8bf9-8711145c648f.png">

**Comment by astoilkov** [comment]
Yes, we are considering it. We are currently experimenting with bringing a preview-like experience in the editor (like images and math). However, if this isn't a successful experiment, we can create a side-by-side preview.

**Comment by darsnack** [comment]
I do like the inline images and equations. Especially the images are very helpful to see as thumbnails for me, because I can just look at it an "read" the image without needing to read the path and remember what image I stored at that path. So, I hope these remain even if a separate preview is considered.

In the same vein, I use the side-by-side preview when the notes become complex to remove any cognitive overhead for me parsing the content. For example, the bulleted list under "Setup" above is much easier to read at a glance in the preview. Of course, inline previews of content in the editor solves this, but at the cost of making editing notes harder. Part of the appeal of editing markdown is that I can see the formatting characters easily. Having to "click" into each element in the editor is a poor writing experience IMO. It works well for only a few elements like images or math blocks. This is where I think the option of a separate preview pane can help without making the editor experience worse.

BTW, this is configurable with the toggle in iA Writer. By default, it works just like Nota, but you can toggle side-by-side preview on as well.

**Comment by astoilkov** [comment]
> I do like the inline images and equations. Especially the images are very helpful to see as thumbnails for me, because I can just look at it an "read" the image without needing to read the path and remember what image I stored at that path. So, I hope these remain even if a separate preview is considered.

Yes, this would remain.

> Part of the appeal of editing markdown is that I can see the formatting characters easily.

I agree, we will probably have a toggle for it.

---

Thanks for the feedback. What you are saying makes sense. We will take those arguments into consideration. Is the side-by-side preview something you are missing a lot in Nota?

**Comment by darsnack** [comment]
> Is the side-by-side preview something you are missing a lot in Nota?

Not really; the inline math helps a lot. The only time I've missed side-by-side is when I'm reviewing/editing a document.
